### ISSUE TITLE
Busybox support

### ISSUE THREAD

**Issue by Rusox89** [aslafy-z/helm-git#108]
Hi, 

First thanks for writing this plugin, works great for me. 

Regarding the last update 0.8.0 I am experiencing issues whilst running on `docker:dind-stable` which uses a BusyBox shell

Here a partial output, 

```
 ls: unrecognized option: I
 BusyBox v1.31.1 () multi-call binary.
 Usage: ls [-1AaCxdLHRFplinshrSXvctu] [-w WIDTH] [FILE]...
 List directory contents
 	-1	One column output
 	-a	Include entries which start with .
 	-A	Like -a, but exclude . and ..
 	-x	List by lines
 	-d	List directory entries instead of contents
 	-L	Follow symlinks
 	-H	Follow symlinks on command line
 	-R	Recurse
 	-p	Append / to dir entries
 	-F	Append indicator (one of */=@|) to entries
 	-l	Long listing format
 	-i	List inode numbers
 	-n	List numeric UIDs and GIDs instead of names
 	-s	List allocated blocks
 	-lc	List ctime
 	-lu	List atime
 	--full-time	List full date and time
 	-h	Human readable sizes (1K 243M 2G)
 	--group-directories-first
 	-S	Sort by size
 	-X	Sort by extension
 	-v	Sort by version
 	-t	Sort by mtime
 	-tc	Sort by ctime
 	-tu	Sort by atime
 	-r	Reverse sort order
 	-w N	Format N columns wide
 	--color[={always,never,auto}]	Control coloring
 Error in plugin 'helm-git': No files have been checked out. Check your Git ref (master) and path (/charts).
```

**Comment by aslafy-z** [comment]
Thanks for the head-up! It should be fixed with #109. Sorry for that regression. Will release ASAP!

**Comment by aslafy-z** [comment]
Published as [`v0.8.1`](https://github.com/aslafy-z/helm-git/releases/tag/v0.8.1)
